libraries/rush-lib/src/logic/operations/PnpmSyncCopyOperationPlugin.ts

```diff
@@ -30,7 +30,11 @@ export class PnpmSyncCopyOperationPlugin implements IPhasedCommandPlugin {
         } = record;
 
         //skip if the phase is skipped or no operation
-        if (status === OperationStatus.Skipped || status === OperationStatus.NoOp) {
+        if (
+          status === OperationStatus.Skipped ||
+          status === OperationStatus.NoOp ||
+          status === OperationStatus.Failure
```

Comment:
@g-chao There are [quite a lot](https://github.com/microsoft/rushstack/blob/4bb48c3d71c3424944d658434ea03a24ae8462ff/libraries/rush-lib/src/logic/operations/OperationStatus.ts#L8) of hooked statuses here. It seems inefficient to invoke `pnpmSyncCopyAsync()` for every single one.  But also, if we accidentally miss an important one, that could produce a difficult-to-repro bug.

Maybe we can replace this `if` with a `switch` statement that explicitly considers every enum member?  Let's deal with that in a separate PR however.  My change here is the minimal fix to eliminate the duplicate error message.